torch/csrc/jit/OVERVIEW.md

```diff
@@ -1295,18 +1298,17 @@ and add an edge from `%output` to `%self`. The alias analysis pass is flow-insen
 As a more involved example, the following TorchScript snippet:
 ```python
 @torch.jit.script
-def foo(a : Tensor, b : Tensor):
-        c = 2 * b
-  a += 1
-  if a.max() > 4:
-    r = a[0]
-  else:
-    r = b[0]
-  return c, r
+def foo(a: Tensor, b: Tensor):
```

Comment:
Also good changes, but IMO would be better in a separate PR given length of this one